static-analysis/pmd-rules/build.gradle

```diff
@@ -18,6 +18,6 @@
 apply from: "${rootDir}/${scriptDir}/standard-subproject-configuration.gradle"
 
 dependencies {
-    testCompile(group: 'net.sourceforge.pmd', name: 'pmd-test', version: '6.11.0')
-    compile(group: 'net.sourceforge.pmd', name: 'pmd-java', version: '6.11.0')
-}
\ No newline at end of file
+    testCompile(group: 'net.sourceforge.pmd', name: 'pmd-test', version: '6.18.0')
```

Comment:
Lets take this opportunity to move this version into our dependency manager, instead of being the sole hard-coded number.